app/lib/common/trxHelper.js

```diff
@@ -186,10 +186,26 @@ function checkBalance(amount, sendAsset, feeAmount, balance) {
     return true;
 }
 
+function shouldPayFeeWithAsset(fromAccount, feeAmount) {
+    if (fromAccount && feeAmount && feeAmount.asset_id === "1.3.0") {
+        const balanceID = fromAccount.getIn([
+            "balances",
+            feeAmount.asset_id
+        ]);
+        const balanceObject = ChainStore.getObject(balanceID);
```

Comment:
The ChainStore is completely synchronous so balanceObject may resolve as undefined even though it's a valid object. Use FetchChain here instead to resolve it, don't include ChainStore in trxHelper.